train-baseline-model.ipynb

```diff
@@ -0,0 +1,928 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Setup"
+   ]
+  },
+  {
```

Comment:
We shouldn't add this file here. Maybe we can have it elsewhere, but not in the `telesto-models` template repository.